charts/kspm-collector/templates/_helpers.tpl

```diff
@@ -134,6 +134,16 @@ Sysdig NATS service URL
 {{- end -}}
 {{- end -}}
 
+### Helper to define if to enable nats_insecure
+{{- define "kspmCollector.natsInsecure" -}}
+{{- if or (.Values.sslVerifyCertificate) (.Values.global.sslVerifyCertificate) -}}
+    "false"
```

Comment:
false here? am I missing something

Reply:
the `nats_insecure` logic works in reverse of the sslVerify. In that if its set to `true` it will work in insecure mode and if `false` it will work in secure mode.

Reply:
I see, that makes sense!